src/components/FeaturePanel/Climbing/Editor/RoutesEditor.tsx

```diff
@@ -125,12 +127,33 @@ export const RoutesEditor = ({
     onMove(positionInImage);
   };
 
+  const preloadOtherPhotos = () => {
+    const photosToLoad = photoPaths.filter((path) => !loadedPhotos[path]);
+
+    const tempLoadedPhotos = photosToLoad.reduce((acc, otherPhotoPath) => {
+      const img = new Image();
+      const url = getCommonsImageUrl(`File:${otherPhotoPath}`, photoResolution);
+      img.src = url;
+
+      return {
+        ...acc,
+        [otherPhotoPath]: {
+          ...acc[otherPhotoPath],
+          [photoResolution]: true,
```

Comment:
Could you name this `photoWidth` instead of `photoResolution`?  Resolution doesn't make that much sense here.

Reply:
testing out notifications 😅